packages/components/src/components/contract-card/contract-card-items/__tests__/vanilla-options-card-body.spec.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { render, RenderResult } from '@testing-library/react';
+import VanillaOptionsCardBody from '../vanilla-options-card-body';
+import { TVanillaOptionsCardBodyProps } from '../vanilla-options-card-body';
+
+describe('VanillaOptionsCardBody', () => {
+  it('should render the correct content for a sold contract', async () => {
+    const { contract_info, currency, getCardLabels, is_sold, progress_slider, status }: TVanillaOptionsCardBodyProps = {
```

Comment:
Mocked props be defined outside of test cases so it can be reused.

const mocked_props = { ... }

See this as an example https://github.com/binary-com/deriv-app/pull/8257/files#diff-3dc84e995751fc2604e3fd882121942026f6839a68c55f8465ccb264301795b7

Reply:
Sure, the props are slightly different for each test case, so I didn't declare them outside but the few common ones can be combined 👍🏻

Reply:
you can change the prop by doing this

contract_info.something.something = 'something'
currency = 'EUR'

inside each test case

Reply:
Yeah, looking to change it now